### PR TITLE
Adds a dark side navigation theme with Menu templatizing and some css tweaks adds logo to landing page.

### PR DESCRIPTION
**Pull request by Phrozyn**
This PR:

- Adds the feature to allow for theme templates to change the navigation menu.
- Adds a dark side-navigation theme
- Fixes some of the text color in top navigation that was unable to be seen when hovered over.
- Fix text color for some buttons on top navigation based themes (attackers page)

### CODE REVIEW

File: meteor/.meteor/versions [hunk 1]
```diff
@@ -98,7 +98,7 @@ templating-compiler@1.3.3
 templating-runtime@1.3.2
 templating-tools@1.1.2
 tracker@1.2.0
-twbs:bootstrap@3.3.6
+twbs:bootstrap@3.3.5
```
Comment (andrewkrug) [inline meteor/.meteor/versions]: @Phrozyn was this an intentional bootstrap downgrade?

File: meteor/client/greeting.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        }
+    } );
+
+    Template.hello.events( {
+        'click': function() {
+            // template data, if any, is available in 'this'
+            Session.set( 'displayMessage', 'Welcome to mozdef.' )
+        }
+    } );
+};
```
Comment (andrewkrug) [inline meteor/client/greeting.js]: Please add new line.

File: meteor/client/loading.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<template name='loading'>
+        loading...
+</template>
```
Comment (andrewkrug) [inline meteor/client/loading.html]: Missing newline.

File: meteor/.meteor/versions (hunk 1 above)
Comment (Phrozyn) [reply meteor/.meteor/versions]: If you look at previous commits you'll see that jeff's commit is what reverted this to an earlier version of bootstrap. This was to facilitate a later version of jquery.

File: meteor/client/greeting.js (hunk 2 above)
Comment (Phrozyn) [reply meteor/client/greeting.js]: done

File: meteor/client/loading.html (hunk 3 above)
Comment (Phrozyn) [reply meteor/client/loading.html]: done

File: meteor/.meteor/versions (hunk 1 above)
Comment (andrewkrug) [reply meteor/.meteor/versions]: ack.  Mostly just wanted to make sure intentional vs NPM mistake.

File: meteor/client/mozdef.js [hunk 4]
```diff
@@ -5,19 +5,13 @@ file, You can obtain one at http://mozilla.org/MPL/2.0/.
...
-import '/client/js/jquery.highlight.js';
-import PNotify from 'pnotify';
 import 'pnotify/dist/pnotify.css';
-import './mozdef.html';
-import './menu.html';
-import './menu.js';
-import '/client/layout.js';
-import '/public/css/dropdowns.css';
+import PNotify from 'pnotify';
+// import './mozdef.html';
```
Comment (pwnbus) [inline meteor/client/mozdef.js]: Should we delete these lines?

Comment (Phrozyn) [reply meteor/client/mozdef.js]: yes, jeff commented these out, was probably due for removal if this PR got approved.

**Comment by Phrozyn** [comment]
I need to fix this PR, I'm going to remove the logo update from it and leave this as a side nav theme only.

File: meteor/client/mozdef.js (hunk 4 above)
Comment (Phrozyn) [reply meteor/client/mozdef.js]: removed in last commit.

**Pull request merged**